index.ts

```diff
@@ -0,0 +1,84 @@
+import express, { Express, Request, Response } from "express";
+import bodyParser from "body-parser";
+import { auctionPaid } from "./types/interfaces";
+import { submitPayment } from "./submitPayment";
+import { saveReciept } from "./saveReceipt";
+import { searchPaid } from "./searchPaid";
+import cors from "cors";
+
+const app: Express = express();
+app.use(bodyParser.json());
+app.use(cors());
+app.use(
+    bodyParser.urlencoded({
+        extended: true,
+    })
+);
+
+const port = 3000;
+
+app.get('/', (req: Request, res: Response) => {
+
+});
+
+app.post(
+    "/submitPayment",
+    async(
+        req: Request<unknown, unknown, unknown, unknown>,
+        res: Response
+    ) =>{
+        try{
+
+            const response = await submitPayment();
+            res.send(response);
+
+        }catch(error){
+            res.sendStatus(400);
+
+        }
+
+    }
+)
+app.post(
```

Comment:
I think instead of having this be a route, just call `saveReceipt()` automatically after `submitPayment()` returns true. Since I don't see the reason of having this be exposed as a route and having the client call it, it just adds an extra step in the chain.